CMakeLists.txt

```diff
@@ -787,13 +787,12 @@ if(BUILD_TESTS)
       historical_queries_test PRIVATE http_parser.host ccf_kv.host
                                       ccf_endpoints.host
     )
-    # Temporarily disabled flaky test
-    # https://github.com/microsoft/CCF/issues/4403 add_unit_test( indexing_test
-    # ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/indexing.cpp
-    # ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/lfs.cpp )
-    # target_link_libraries( indexing_test PRIVATE ccf_endpoints.host sss.host
-    # ccf_kv.host ) target_link_libraries( indexing_test PRIVATE
-    # ccf_endpoints.host ccf_kv.host )
+    add_unit_test(
+      indexing_test ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/indexing.cpp
+      ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/lfs.cpp
+    )
+    target_link_libraries(indexing_test PRIVATE ccf_endpoints.host ccf_kv.host)
+    target_link_libraries(indexing_test PRIVATE ccf_endpoints.host ccf_kv.host)
```

Comment:
```suggestion
```

Not sure how we ended up with duplicate lines here

Reply:
Bad merge, thanks!